src/platform/packages/shared/kbn-rrule/rrule.ts

```diff
@@ -74,19 +32,15 @@ type AllResult = Date[] & {
 };
 
 const ALL_LIMIT = 10000;
+const TIMEOUT_LIMIT = 100000;
```

Comment:
Just a drive-by comment wondering if we really need to let it run 100,000 iterations before hard-exiting?

Reply:
I am not familiar with this stuff at all but seems like if we could lower this and have it exit earlier, would be better. But if we need up to 100,000 iterations, then you can disregard :)

Reply:
If the default limit on getting "all" recurrences of a schedule that has no end is 10k, we want to give it enough of a buffer that it might actually be able to find 10k recurrences.

Kibana isn't really hitting those limits right now but since 100k iterations takes a negligible amount of compute time to do, it seems like a decent ceiling to future-proof this.